sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
+
+#define MAX_COUNTERS 100
+#define COUNTER_LENGTH 32
+#define INT int
+
+#ifdef SOKUTEI_NO_STDLIB
+#ifndef SOKUTEI_STDLIB
+#define SOKUTEI_STDLIB
+
+int sokutei_strcmp(const char *a, const char *b){
+    int i = 0;
+    while(a[i] && b[i] && a[i] == b[i]) i++;
+    return a[i] - b[i];
+}
+#define STRCMP(a, b) sokutei_strcmp(a, b)
+
+char *sokutei_strcpy(char *a, const char *b){
+    int i;
+    for(i = 0; a[i] = b[i]; i++);
```

Comment:
Consider the following scenario: The maximum length of counter names is set to `32`. The user passes in a string of `33` letters and we use this `strcpy` implementation. What is going to be the result?

Hint: [strcpy_s](https://en.cppreference.com/w/c/string/byte/strcpy).